lib/casestatus.ts

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
```

Comment:
Could we rename this file to `lib/caseStatus.ts` to follow our naming convention? Similarly for the spec file.

Reply:
done